DOC-842: Added release note for context toolbar changes (#2063)
tinymce/tinymce-docs: release-notes/release-notes59.md

The following enhancements were made for the {{site.productname}} 5.9 release.

> ## Context Toolbars

Reviewer: ```suggestion
### Context Toolbars
```
Should it be this way around?

Reviewer: Tyler has already addressed this in his own suggestion, that adds more improvements.

> ## Context Toolbars

Author: I wanted to avoid mentioning `positioning` in the heading as this also includes the transitioning. Maybe something like this instead?
```suggestion
### Improved Context Toolbars user experience
```

> {{site.productname}} 5.9 improves context toolbar positioning so that the toolbar will remain in the previous position while scrolling and now moves out of the way if the toolbar will overlap the content. For example, previously when using a table the context toolbar may have covered the cells in the first row. Now, when clicking into any cell within that row, the toolbar will move to the bottom of the viewport instead.

Reviewer 2: >move to the bottom of the viewport instead

It moves to the bottom of the table right?

Author: No, it's the bottom of the viewport (it docks to the bottom, as the bottom of the table is out of view).

> {{site.productname}} 5.9 improves context toolbar positioning so that the toolbar will remain in the previous position while scrolling and now moves out of the way if the toolbar will overlap the content. For example, previously when using a table the context toolbar may have covered the cells in the first row. Now, when clicking into any cell within that row, the toolbar will move to the bottom of the viewport instead.

Reviewer 3: ```suggestion
{{site.productname}} 5.9 improves context toolbar positioning by ensuring the toolbar remains in the same position when the user scrolls and repositions if the toolbar will overlap the selected content or the cursor. For example: Prior to this improvement, if a user was editing a table, the context toolbar may have covered cells in the first row while it was being edited. Now, when clicking into any cell within that row, the toolbar will move to the bottom of the viewport instead.
```

Author: The first sentence no longer runs together in this suggestion, so it's confusing?

Author: ```suggestion
{{site.productname}} 5.9 improves context toolbar positioning by ensuring the toolbar remains in the same position when the user scrolls. Additionally, the context toolbar repositions if the toolbar will overlap the selected content or the cursor. For example: Prior to this improvement, if a user was editing a table, the context toolbar may have covered cells in the first row while it was being edited. Now, when clicking into any cell within that row, the toolbar will move to the bottom of the viewport instead.
```
Maybe?

Reviewer 3: thanks. yes. use your rework of my rework pls

> To further enhance the user experience, the context toolbar will now use a short animation when the toolbar transitions from above or below an element.

Reviewer 3: ```suggestion
To further enhance the user experience, the context toolbar will now use a short animation when the toolbar transitions between above and below an element.
```

Author: This suggestion doesn't make sense sorry.

Reviewer 3: > transitions around an element.

?

Author: ```suggestion
To further enhance the user experience, the context toolbar will now use a short animation when the toolbar transitions between different locations.
```
